### ISSUE TITLE
New-PnPAzureCertificate: The FriendlyName value cannot be set on Unix.

### ISSUE THREAD

**Issue by coreyroth** [pnp/powershell#44]
### Reporting an Issue or Missing Feature
When I execute New-PnPAzureCertificate on Mac OS Big Sur, I get an error stating that the FriendlyName cannot be set on Unix.

`New-PnPAzureCertificate -OutPfx "pnp.pfx" -ValidYears 1 -CertificatePassword $secPassword`

I haven't done much with these certificates before, but from everything I have read, i think I am doing it right.

### Expected behavior 
Output of a pfx file.

### Actual behavior 
The following error occurs on Mac OS Big Sur:
_New-PnPAzureCertificate: The FriendlyName value cannot be set on Unix._

### Steps to reproduce behavior
Please include complete code samples in-line or linked from [gists](https://gist.github.com/)

```
$Password = "mypassword"
$secPassword = ConvertTo-SecureString -String $Password -AsPlainText -Force
New-PnPAzureCertificate -OutPfx "pnp.pfx" -ValidYears 1 -CertificatePassword $secPassword
```

### What is the version of the Cmdlet module you are running?
(you can retrieve this by executing ```Get-Module -Name sharepointpnppowershell* -ListAvailable```)
PnP.PowerShell 0.3.3

**Comment by erwinvanhunen** [comment]
Thanks for reporting!

It's fixed and it will be available in nightly build of tomorrow (0.3.5).

**Comment by coreyroth** [comment]
Thanks Erwin!  It's always a relief to know it's a real issue and not user error on my part.  I'll give it a try in a couple of days.
